app/core/CocoVueRouter.js

```diff
@@ -0,0 +1,32 @@
+import VueRouter from 'vue-router'
```

Comment:
Why is this called `CocoVueRouter.js` instead of `VueRouter.js`?

Reply:
The backbone router is actually a `CocoRouter` in `Router.coffee` so I decided to follow the class name pattern (but not rename the file in case tracing down Router loading took a lot of time).  I think we should align the names and I like VueRouter - so maybe we do VueRouter and rename CocoRouter later?

Reply:
Yea, can rename this one to VueRouter if convenient and BackboneRouter later.

Reply:
Updated